### ISSUE TITLE
Broken link in keycap documentation

### ISSUE THREAD

**Issue by whodevil** [tmk/tmk_keyboard#202]
Hello,

There is a broken link in https://github.com/tmk/tmk_keyboard/blob/master/doc/keymap.md

```
 In regard to implementation side most of keycodes are identical with [HID usage][HID_usage](pdf) sent to host for real and some virtual keycodes are defined to support special actions.
[HID_usage]: http://www.usb.org/developers/devclass_docs/Hut1_11.pdf
```

**Comment by tmk** [comment]
I'll fix later. Thanks.

new location:
http://www.usb.org/developers/hidpage/Hut1_12v2.pdf

**Comment by tmk** [comment]
Fixed at 3f32b224788f9e938e41e02f6ad6fcbbec67479f
